[2025-08-16 21:25:29,291] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.drop_table_business_types manual__2025-08-16T21:25:27.221750+00:00 [queued]>
[2025-08-16 21:25:29,299] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.drop_table_business_types manual__2025-08-16T21:25:27.221750+00:00 [queued]>
[2025-08-16 21:25:29,304] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-16 21:25:29,304] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-16 21:25:29,305] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-16 21:25:29,317] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): drop_table_business_types> on 2025-08-16 21:25:27.221750+00:00
[2025-08-16 21:25:29,323] {standard_task_runner.py:52} INFO - Started process 322 to run task
[2025-08-16 21:25:29,333] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', '1_load_initial_data', 'drop_table_business_types', 'manual__2025-08-16T21:25:27.221750+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/initialise_data.py', '--cfg-path', '/tmp/tmp2ccxxfh4', '--error-file', '/tmp/tmpgcmj2f9d']
[2025-08-16 21:25:29,336] {standard_task_runner.py:80} INFO - Job 5: Subtask drop_table_business_types
[2025-08-16 21:25:29,440] {task_command.py:369} INFO - Running <TaskInstance: 1_load_initial_data.drop_table_business_types manual__2025-08-16T21:25:27.221750+00:00 [running]> on host 6a6e6aafe640
[2025-08-16 21:25:29,554] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=1_load_initial_data
AIRFLOW_CTX_TASK_ID=drop_table_business_types
AIRFLOW_CTX_EXECUTION_DATE=2025-08-16T21:25:27.221750+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-08-16T21:25:27.221750+00:00
[2025-08-16 21:25:29,556] {sql.py:262} INFO - Executing: DROP TABLE IF EXISTS dbt_raw_data.business_types;
[2025-08-16 21:25:29,583] {base.py:68} INFO - Using connection ID 'dbt_postgres_instance_raw_data' for task execution.
[2025-08-16 21:25:29,606] {logging_mixin.py:115} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/connection.py:339 DeprecationWarning: The "schema" arg has been renamed to "database" as it contained the database name.Please use "database" to set the database name.
[2025-08-16 21:25:29,625] {base.py:68} INFO - Using connection ID 'dbt_postgres_instance_raw_data' for task execution.
[2025-08-16 21:25:29,636] {sql.py:375} INFO - Running statement: DROP TABLE IF EXISTS dbt_raw_data.business_types;, parameters: None
[2025-08-16 21:25:29,662] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=1_load_initial_data, task_id=drop_table_business_types, execution_date=20250816T212527, start_date=20250816T212529, end_date=20250816T212529
[2025-08-16 21:25:29,709] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-16 21:25:29,774] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
